sphinx_pyscript/__init__.py

```diff
@@ -72,52 +78,66 @@ def run(self):
             code = "\n".join(self.content)
         else:
             raise self.error("Must provide either content or the 'file' option")
-        return [nodes.raw("", f"<py-script>\n{code}\n</py-script>\n", format="html")]
+        return [
+            nodes.raw("", f"<script type='py'>\n{code}\n</script>\n", format="html")
+        ]
 
 
-class PyRepl(SphinxDirective):
-    """Add a py-repl tag"""
+class PyEditor(SphinxDirective):
+    """Add a py-editor tag"""
 
     has_content = True
     option_spec = {
-        "auto-generate": directives.flag,
-        "output": directives.unchanged,
+        "setup": directives.flag,
+        "env": directives.unchanged,
+        "config": directives.unchanged,
     }
 
     def run(self):
-        """Add the py-repl tag"""
+        """Add the py-editor tag"""
         attrs = ""
         code = ""
-        if "auto-generate" in self.options:
-            attrs += ' auto-generate="true"'
-        if "output" in self.options:
-            attrs += f' output="{self.options["output"]}"'
+        if "env" in self.options:
+            attrs += f' env="{self.options["""env"""]}"'
+        if "config" in self.options:
+            attrs += f' config="{self.options["""config"""]}"'
+        if "setup" in self.options:
+            attrs += "setup"
         if self.content:
             code = "\n".join(self.content)
-        return [nodes.raw("", f"<py-repl{attrs}>\n{code}\n</py-repl>\n", format="html")]
+        return [
+            nodes.raw(
+                "",
+                f"<script type='py-editor' {attrs}>\n{code}\n</script>\n",
+                format="html",
+            )
+        ]
 
 
 class PyTerminal(SphinxDirective):
     """Add a py-terminal tag"""
 
-    option_spec = {
-        "auto": directives.flag,
-    }
+    option_spec = {}
 
     def run(self):
         """Add the py-terminal tag"""
         attrs = ""
-        if "auto" in self.options:
-            attrs += " auto"
-        return [nodes.raw("", f"<py-terminal{attrs}></py-terminal>\n", format="html")]
+        if "worker" in self.options:
```

Comment:
you haven't added an option for this in the `option_spec`

Reply:
Corrected.